src/audio/kpb.c

```diff
@@ -91,6 +91,10 @@ struct comp_data {
 #ifdef CONFIG_IPC_MAJOR_4
 	struct ipc4_kpb_module_cfg ipc4_cfg;
 #endif /* CONFIG_IPC_MAJOR_4 */
+	uint32_t num_of_sel_mic;
```

Comment:
Please dont include the rimage update in this patch, if its needed for this feature then it needs to be a west update.

Reply:
Fixed